vpr/src/device/rr_spatial_lookup.h

```diff
@@ -102,27 +111,27 @@ class RRSpatialLookup {
 
     /* -- Mutators -- */
   public:
-    /**
-     * Reserve the memory for a list of nodes at (x, y) location with given type and side
-     */
+    /** @brief Reserve the memory for a list of nodes at (x, y) location with given type and side */
     void reserve_nodes(int x,
                        int y,
                        t_rr_type type,
                        int num_nodes,
                        e_side side = SIDES[0]);
 
     /**
-     * Register a node in the fast look-up 
-     * - You must have a valid node id to register the node in the lookup
-     * - (x, y) are the coordinate of the node to be indexable in the fast look-up
-     * - type is the type of a node
-     * - ptc is a feature number of a node, which can be
-     *   - the class number of a common SINK/SOURCE node of grid, 
-     *   - pin index in a tile when type is OPIN/IPIN
-     *   - track index in a routing channel when type is CHANX/CHANY
-     * - side is the side of node on the tile, applicable to OPIN/IPIN 
-     *
-     * Note that a node added with this call will not create a node in the rr_graph_storage node list
+     * @brief Register a node in the fast look-up 
+     *
+     * @note You must have a valid node id to register the node in the lookup
+     *
+     *   @param (x, y) are the coordinate of the node to be indexable in the fast look-up
+     *   @param type is the type of a node
+     *   @param ptc is a feature number of a node, which can be
+     *     - the class number of a common SINK/SOURCE node of grid, 
+     *     - pin index in a tile when type is OPIN/IPIN
+     *     - track index in a routing channel when type is CHANX/CHANY
+     *   @param side is the side of node on the tile, applicable to OPIN/IPIN 
+     *
+     * @note a node added with this call will not create a node in the rr_graph_storage node list
      * You MUST add the node in the rr_graph_storage so that the node is valid  
      *
      * TODO: Consider to try to return a reference to *this so that we can do chain calls
```

Comment:
Maybe the TODO comment should be outside the doxygen commenting (put as just a regular comment) so it doesn't show up in the API comment?